### DISCUSSION TITLE
Bindings for storybook : a suggestion

### DISCUSSION THREAD

**Discussion by halcwb** [SAFE-Stack/SAFE-template#594] [Ideas]
A while ago I discussed with @Zaid-Ajaj the possibility of porting the wonderful [storybook](https://storybook.js.org/) to Fable.React and/or Feliz. This has been done [before](https://github.com/gamurs-group/fable-storybook), but this is a while ago.

The idea is that you can spin up you're storybook and start experimenting and creating isolated components. This is the way I develop my SPA's nowadays, with lots of isolated [components](https://github.com/halcwb/PICEDashboard/tree/master/src/Client/Components). Ideally I would like to create these components in a separate storybook. 

Also, the online [fable repl](https://fable.io/repl/) could have such a storybook to experiment with different components.

The ability to have a SAFE-template with such a storybook would be enormously helpful for both newbies and 'oldbies'. I have to admit that I do not have the expertise to do this myself. But I honestly believe that this would be something of great value for the SAFE stack. 

What do you think?

**Comment by theimowski** [comment]
I haven't yet had chance to use Storybook, so hard to judge for me, but surely the more bindings we have for Fable the better

**Comment by halcwb** [comment]
I think it really would help promoting Fable and the SAFE.Stack. Unfortunately I lack expertise and time to do it myself. I am looking for funding though.
